cpp/daal/src/services/env_detect.cpp

```diff
@@ -156,7 +163,6 @@ DAAL_EXPORT void daal::services::Environment::initNumberOfThreads()
 DAAL_EXPORT daal::services::Environment::~Environment()
 {
     daal::services::daal_free_buffers();
-    _daal_tbb_task_scheduler_free(_globalControl);
```

Comment:
can you please add TODO comment here

Reply:
What kind of TODO comment do you mean?

Reply:
@Alexandr-Solovev Please provide more details about what TODO is needed for.
I would suggest covering this with tickets instead of adding TODO comment